app/assets/src/components/views/phylo_tree/PhyloTreeCreation.jsx

```diff
@@ -480,16 +458,12 @@ class PhyloTreeCreation extends React.Component {
           <div className="wizard__page-2__searchbar">
             <div className="wizard__page-2__searchbar__container">Organism</div>
             <div className="wizard__page-2__searchbar__container">
-              {this.state.taxaLoaded ? (
-                <SearchBox
-                  source={this.state.taxonList}
-                  onResultSelect={this.handleSelectTaxon}
-                  initialValue={this.state.taxonName}
-                  placeholder="Genus name"
-                />
-              ) : (
-                <LoadingIcon />
```

Comment:
LoadingIcon shows up when the *search box* is not yet loaded... because we used to transfer all the taxa to the frontend, which meant it took a while for the search box to show up. Now, since we don't preload any of the data, the search box will always show up immediately, so no need for LoadingIcon. Other search boxes that don't preload data also don't have a loading icon

Reply:
understood